app/models/job.rb

```diff
@@ -1,4 +1,37 @@
 class Job < ActiveRecord::Base
   validates :title, :level_of_interest, :city, presence: true
+  validates :category_id, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments, dependent: :destroy
+
+  def self.payload
```

Comment:
i like your thought process here - but i'd encourage you to choose a different name - also does this belong in this particular model? or maybe do we create a PORO that handles the formatting of this data?